components/api/Cargo.toml

```
@@ -0,0 +1,32 @@
+[package]
+name = "exonum-api"
+version = "0.13.0-rc.2"
+edition = "2018"
+authors = ["The Exonum Team <exonum@bitfury.com>"]
```

An email address has been changed
```suggestion
authors = ["The Exonum Team <contact@exonum.com>"]
```